core/src/main/scala/com/softwaremill/session/RefreshTokenStorage.scala

```diff
@@ -66,4 +66,10 @@ trait InMemoryRefreshTokenStorage[T] extends RefreshTokenStorage[T] {
   }
 
   def log(msg: String): Unit
-}
\ No newline at end of file
+}
+
+/**
+ * Can't use the trait above in Java code, hence this wrapper
+ * http://stackoverflow.com/questions/7637752/using-scala-traits-with-implemented-methods-in-java
+ */
+abstract class InMemoryRefreshTokenStorageWrapper[T]() extends InMemoryRefreshTokenStorage[T]
```

Comment:
maybe move this to the `javadsl` package? then it can have a name without the -wrapper suffix as well